src/core/constants.ts

```diff
@@ -6,6 +6,25 @@ export const EXPERIMENTS =
+export const SUPPORTED_CAPABILITIES = [
```

Comment:
We already have a place for this - 
https://github.com/dilame/instagram-private-api/blob/v1.0.0/src/samples/supported-capabilities.json